suricata.yaml.in

```diff
@@ -200,6 +200,38 @@ outputs:
             # the example below adds three additional fields when uncommented
             #custom: [Accept-Encoding, Accept-Language, Authorization]
         - dns:
+            # Use version 2 logging with the new format:
+            # dns answers will be logged in one single event
+            # rather than an event for each of it.
+            # Without setting a version the version
+            # will fallback to 1 for backwards compatibility.
+            version: 2
+
+            # Enable/disable this logger. Default: enabled.
+            #enabled: no
+
+            # Control logging of requests and responses:
+            # - requests: enable logging of DNS queries
+            # - responses: enable logging of DNS answers
+            # By default both requests and responses are logged.
+            #requests: no
+            #responses: no
+
+            # Format of answer logging:
+            # - detailed: array item per answer
+            # - grouped: answers aggregated by type
+            # Default: all
+            #answer-format: [detailed, grouped]
+
+            # Answer types to log.
+            # Default: all
+            #answer-types: [a, aaaa, cname, mx, ns, ptr, txt]
+        - dns:
```

Comment:
I may be wrong but we have two **- dns:** sections then which might not work or at least lead to confusion?

Reply:
Its perfectly valid. Even now you can duplicate the "dns" section of eve, or another one, and you'll get 2 entries for everything logged as it registers another logger.

The idea here is that you get the "v2" style output by default with a new config, but an old config will get the old style. Or if you really want, you can have both enabled at the same time.

I agree on the confusing part. I wonder if it would make sense to remove the old one from the default configuration, but leave it in the docs?

Reply:
I think it's better to leave it because an user could easily switch to version 1.
We may comment it and explaining the purpose of that dns section, to avoid confusion.

Reply:
What about this pont?

Reply:
I think the new 'v2' format should be the only one in the default yaml. But there should be a link to the docs for the v1 format.

I think we'll start deprecating the v1 soon, so lets push ppl to v2.